### PR TITLE
Added new Github sample for KB

### PR DESCRIPTION
**Pull request by VijayadharshiniMathiyalagan**
Description:
Added new Github sample for the below

- Replace-bookmark-content-with-HTML
- Set-text-right-to-stacked-bar-chart
- Remove-content-control-not-content
- Add-DocVariable-in-DisplayBarCode
- Add-line-break-after-replace-text
- Copy-header-footer-to-other-sections
- Set-image-in-first-page-header
- Reduce-height-of-empty-cells
- table/Remove-contents-above-table
- Add-new-page-at-end/Add-new-page-at-end

### CODE REVIEW

**Review by LokeshBaskar — APPROVED** [review]
No concerns

File: Sections/Set-Image-in-First-page-Header/.NET/Set-Image-in-First-page-Header/Set-Image-in-First-page-Header.csproj
Comment (LokeshBaskar) [inline Sections/Set-Image-in-First-page-Header/.NET/Set-Image-in-First-page-Header/Set-Image-in-First-page-Header.csproj:1]: Please move the sample under .NET folder

**Review by LokeshBaskar — APPROVED** [review]

File: Word-to-PDF-Conversion/Replace-Embedded-Excel-as-Image/Replace-Embedded-Excel-as-Image/Program.cs [hunk 1]
```diff
@@ -24,13 +21,9 @@
 FileStream outputStream = new FileStream(Path.GetFullPath(@"Output/Output.pdf"), FileMode.Create, FileAccess.Write);
 // Save the generated PDF to the specified file stream
 pdf.Save(outputStream);
```
Comment (LokeshBaskar) [inline Word-to-PDF-Conversion/Replace-Embedded-Excel-as-Image/Replace-Embedded-Excel-as-Image/Program.cs:23]: Please remove the setting options
docIORenderer.Settings = new DocIORendererSettings();

And also use "using" statement for File stream open and save

**Review by LokeshBaskar — CHANGES_REQUESTED** [review]

File: Paragraphs/Apply-indentation-for-import-content/.NET/Apply-indentation-for-import-content/Program.cs [hunk 2]
```diff
@@ -27,6 +27,10 @@
 FileStream outputStream = new FileStream(Path.GetFullPath(@"Output/Output.docx"), FileMode.Create, FileAccess.Write);
 mainDocument.Save(outputStream, FormatType.Docx);
 
+//Dispose the streams.
```
Comment (Suriya-Balamurugan) [inline Paragraphs/Apply-indentation-for-import-content/.NET/Apply-indentation-for-import-content/Program.cs:30]: Add proper using statement to close file stream and Word document instance

File: Paragraphs/Apply-indentation-for-import-content/.NET/Apply-indentation-for-import-content/Program.cs [hunk 3]
```diff
@@ -27,6 +27,10 @@
...
+fileStream1.Dispose();
+fileStream.Dispose();
+
```
Comment (Suriya-Balamurugan) [inline Paragraphs/Apply-indentation-for-import-content/.NET/Apply-indentation-for-import-content/Program.cs:33]: Use proper variable names and modify proper input document name.

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: Sections/Set-Image-in-First-page-Header/.NET/Set-Image-in-First-page-Header/Set-Image-in-First-page-Header.csproj [hunk 4]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net8.0</TargetFramework>
-    <RootNamespace>Set_image_in_first_page_only</RootNamespace>
+    <RootNamespace>Set-Image-in-First-page-Header</RootNamespace>
```
Comment (Suriya-Balamurugan) [inline Sections/Set-Image-in-First-page-Header/.NET/Set-Image-in-First-page-Header/Set-Image-in-First-page-Header.csproj:6]: Modify casing of the sample name

Set_image_in_first_page_header

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: Paragraphs/Apply-indentation-for-import-content/.NET/Apply-indentation-for-import-content/Program.cs [hunk 5]
```diff
@@ -1,59 +1,71 @@
...
+using (FileStream destinationStream = new FileStream(Path.GetFullPath(@"Data/DestinationDocument.docx"), FileMode.Open))
+{
+    // Load the main Word document.
+    WordDocument destinationDocument = new WordDocument(destinationStream, FormatType.Docx);
+
+    // Open the existing source document from a file stream.
+    using (FileStream sourceStream = new FileStream(Path.GetFullPath(@"Data/SourceDocument.docx"), FileMode.Open))
+    {
+        // Load the source Word document.
+        WordDocument sourceDocument = new WordDocument(sourceStream, FormatType.Docx);
```
Comment (Suriya-Balamurugan) [inline Paragraphs/Apply-indentation-for-import-content/.NET/Apply-indentation-for-import-content/Program.cs:14]: Use using statement for Word document instances

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: Paragraphs/Apply-indentation-for-import-content/.NET/Apply-indentation-for-import-content/Program.cs (hunk 2 above)
Comment (VijayadharshiniMathiyalagan) [reply Paragraphs/Apply-indentation-for-import-content/.NET/Apply-indentation-for-import-content/Program.cs]: addressed

File: Paragraphs/Apply-indentation-for-import-content/.NET/Apply-indentation-for-import-content/Program.cs (hunk 3 above)
Comment (VijayadharshiniMathiyalagan) [reply Paragraphs/Apply-indentation-for-import-content/.NET/Apply-indentation-for-import-content/Program.cs:43]: Addressed

File: Sections/Set-Image-in-First-page-Header/.NET/Set-Image-in-First-page-Header/Set-Image-in-First-page-Header.csproj (hunk 4 above)
Comment (VijayadharshiniMathiyalagan) [reply Sections/Set-Image-in-First-page-Header/.NET/Set-Image-in-First-page-Header/Set-Image-in-First-page-Header.csproj]: Addressed

File: Content-Controls/Remove-content-control-not-content/.NET/Remove-content-control-not-content/Program.cs [hunk 6]
```diff
@@ -0,0 +1,70 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+
+using (FileStream fileStreamPath = new FileStream(Path.GetFullPath(@"Data/Input.docx"), FileMode.Open, FileAccess.Read))
+{
+    //Load the Word document from the FileStream.
+    WordDocument document = new WordDocument(fileStreamPath, FormatType.Docx);
```
[the remark this answers is not in the document]
Comment (VijayadharshiniMathiyalagan) [reply Content-Controls/Remove-content-control-not-content/.NET/Remove-content-control-not-content/Program.cs]: Addressed

File: Word-document/Add_new_page_at_end/.NET/Add_new_page_at_end.sln [hunk 7]
```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.12.35309.182
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Add_new_page_at_end", "Add_new_page_at_end\Add_new_page_at_end.csproj", "{998C35CE-53D4-483D-9D2F-C9C3E712C0CC}"
```
[the remark this answers is not in the document]
Comment (VijayadharshiniMathiyalagan) [reply Word-document/Add_new_page_at_end/.NET/Add_new_page_at_end.sln]: Addressed

File: Sections/Set-Image-in-First-page-Header/.NET/Set-Image-in-First-page-Header/Set-Image-in-First-page-Header.csproj
Comment (VijayadharshiniMathiyalagan) [reply Sections/Set-Image-in-First-page-Header/.NET/Set-Image-in-First-page-Header/Set-Image-in-First-page-Header.csproj:1]: Addressed

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**